src/components/CustomerProfile.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+
+function CustomerProfile(props) {
+  const [customerData, setCustomerData] = useState(null);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${props.customerId}`)
```

Comment:
Really nice and clean component! The only thing I would suggest is to practice destructuring props